### PR TITLE
wip: storyshot

### PR DESCRIPTION
**Pull request by janehughes**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// See https://jestjs.io/docs/en/configuration.
+module.exports = {
```
Comment (thibaudcolas) [inline jest.config.js:2]: As we discussed this will need updating to be based on the existing Jest config in the `package.json`.

**Review by thibaudcolas — COMMENTED** [review]
Adding a few comments for things we didn’t discuss in person.

**Comment by zerolab** [comment]
@janehughes @thibaudcolas is this something that you will be working on or shall we close it as there was no progress in a couple of years?

**Comment by thibaudcolas** [comment]
@zerolab fair enough, we do want this but there’s #8660 that’s quite well sign-posted.

**Pull request closed without merging**